Sage/carbon
# fix(form): sticky footer form retains focus on page when tabbing FE-2498

### Proposed behaviour
User should be able to Tab/Shift+Tab through focusable elements on the page including Form Component and child elements while Form Component has enabled sticky footer.
User should be able to trigger form validation through clicking Save/Submit button and regain focus using Tab/Shift+Tab keys.

### Current behaviour
For Form Component with sticky footer enabled, the focus is lost when tabbing beyond last form element (for example submit/save button). Also focus is lost when Submit/Save button is clicked but form is not submitted due to validation. Focus is never regain using keyboard alone.

### Checklist

<del>- [ ] Unit tests
<del>- [ ] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Testing instructions
Storybook -> Experimental -> Form -> default (story)
Select sticky footer (knob)
Select form element to gain focus, then use Tab Key to change focus until Save button is focused, use Enter key, this should cause Save button to loose focus, then either, pressing Tab or Shift+Tab should focus element on the page.
